bom/contrib/jaxb/pom.xml

```diff
@@ -0,0 +1,41 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?><project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <groupId>org.openhab.core.contrib</groupId>
+  <artifactId>jaxb</artifactId>
+  <version>2.5.0-SNAPSHOT</version>
+  <packaging>pom</packaging>
+  
+  <name>openHAB Core :: BOM :: Contrib JAXB</name>
+  <description>The dependencies are used to provide JAXB bundles</description>
+
+  <!-- do not upgrade to 2.3 as long as Java 8 support is needed -->
+  
+  <dependencies>
+    <dependency>
```

Comment:
Could we drop the provided scope on all dependencies here?
If the default / compile is used here and you depend on that POM using provided, all transitive ones (so this one) will be provided.
If provided is used here, you cannot change it using compile for the POM, it remains provided.
So, we should leave it to the usage and not limit the scope already.